db/migrate/20211217184706_add_session_token_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddSessionTokenToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :session_token, :string, null: false
+    add_index :users, :session_token, unique: true
```

Comment:
Always ensure there's a unique value since this will serve as a way to identify a user.